go/vt/tabletserver/tx_pool.go

```diff
@@ -171,12 +172,24 @@ func (axp *TxPool) Begin(ctx context.Context) (int64, error) {
 }
 
 // Commit commits the specified transaction.
-func (axp *TxPool) Commit(ctx context.Context, transactionID int64, messager *MessagerEngine) error {
+func (axp *TxPool) Commit(ctx context.Context, transactionID int64, messager *MessagerEngine, schemaEngine *schema.Engine) error {
```

Comment:
This change shouldn't be needed. We should commit & reload schema in QueryExecutor instead (if we still want to allow this).